nixos/modules/services/hardware/logiops.nix

```diff
@@ -0,0 +1,164 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mdDoc mkEnableOption mkIf mkOption types;
+  cfg = config.services.keyd;
```

Comment:
Is this correct? I would've expected it to say `cfg = config.services.logiops`

Reply:
Ah sorry for the confusion, I must've missed this when copying over some code from a different service's setup.